src/index.js

```diff
@@ -1,30 +1,32 @@
-const https = require('https');
+const { fetch } = require('undici');
 
 const MANIFEST_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/manifest.json';
+const CONSOLE_README_SUFFIX_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/consoleReadmeSuffix.md';
+const LOCAL_README_SUFFIX_URL = 'https://raw.githubusercontent.com/apify/actor-templates/master/templates/localReadmeSuffix.md';
 
 exports.fetchManifest = async () => {
-    return new Promise((resolve, reject) => {
-        https.get(MANIFEST_URL, (res) => {
-            let json = '';
-            res
-                .on('data', (chunk) => {
-                    json += chunk;
-                })
-                .on('end', () => {
-                    if (res.statusCode === 200) {
-                        try {
-                            const data = JSON.parse(json);
-                            resolve(data);
-                        } catch (e) {
-                            reject(e);
-                        }
-                    } else {
-                        reject(new Error(`Status: ${res.statusCode}\n${json}`));
-                    }
-                })
-                .on('error', (err) => reject(err));
-        }).on('error', (err) => reject(err));
-    });
+    try {
+        const manifestResponse = await fetch(MANIFEST_URL);
+
+        if (manifestResponse.status !== 200) throw new Error(`Could not fetch manifest from ${MANIFEST_URL}`);
+
+        const manifestData = await manifestResponse.json();
+        const consoleResponse = await fetch(CONSOLE_README_SUFFIX_URL);
+        const localResponse = await fetch(LOCAL_README_SUFFIX_URL);
+
+        if (consoleResponse.status === 200) {
+            manifestData.consoleReadmeSuffix = await consoleResponse.text();
+        }
+
+        if (localResponse.status === 200) {
+            manifestData.localReadmeSuffix = await localResponse.text();
+        }
+
+        return manifestData;
+    } catch (err) {
+        console.error(err);
+        throw err;
+    }
```

Comment:
Is this needed? Wouldn't it only print the error twice and behave the same as without the try catch block?